superflore/test_integration/gentoo/build_base.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Open Source Robotics Foundation, Inc.
```

Comment:
This whole file is not used yet?

Reply:
Not at the moment. My plan was to use it in a follow up PR (since we're many commits in at this point).

But I suppose I can start the `superflore-check-ebuilds` in this PR.